Readers/ReaderFactory.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Ovation.FasterQC.Net.Utils;
+
+namespace Ovation.FasterQC.Net.Readers
+{
+    public static class ReaderFactory
+    {
+        public static ISequenceReader Create(CliOptions settings)
+        {
+            return settings switch
+            {
+                { Fastq: true } => new FastqLineReader(settings.InputFilename, true),
```

Comment:
shouldn't this 'true' use the gzipped option?

Reply:
oops, yep